graylog2-server/src/main/java/org/graylog2/plugin/security/RestPermissionsPlugin.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.plugin.security;
+
+import java.util.Set;
+
+public interface RestPermissionsPlugin {
```

Comment:
I think this class name is a little bit misleading. It's not a plugin, but it's a set of permissions provided by a plugin. So `PluginPermissions` or `PluginRestPermissions` would probably make more sense.